scripts/postprocessing/infogain_selector.py

```diff
@@ -0,0 +1,98 @@
+# -*- coding: utf-8 -*-
+import numpy as np
+from scipy import log2
+from scipy.integrate import nquad
+from scipy.special import gammaln, psi
+from scipy.stats import dirichlet
+
+
+def make_range(*x):
+    """
+    constructs leftover values for the simplex given the first k entries
+    (0,x_k) = 1-(x_1+...+x_(k-1))
+    """
+    return (0, max(0, 1 - sum(x)))
+
+
+def relative_entropy(p, q):
```

Comment:
A few `np.array` type-annotations would be nice in this file, too .. but we can do this later.

Reply:
Sure, I didn't spend too much time prettying up the numerical solver, since I was able to find an analytic solution which is way faster and not numerically horrible.
At the moment I have the numerical solver only here because I needed to do some additional tests, but technically, we can remove all the numerical stuff.

Reply:
OK, fair. I don't want to discourage you from checking in algo-development-code - this work is very important for the project (you can use `postprocessing` as some kind of prototyping area). Dense math heavy code is in general different from the 'verbose' frontend code and other aspects matter for readability and overall solution-quality.